raiden/network/pathfinding.py

```diff
@@ -148,8 +148,8 @@ def get_pfs_info(url: str) -> Optional[PFSInfo]:
             operator=infos["operator"],
             version=infos["version"],
         )
-    except (json.JSONDecodeError, requests.exceptions.RequestException, KeyError):
-        return None
+    except (json.JSONDecodeError, requests.exceptions.RequestException, KeyError) as e:
+        raise ServiceRequestFailed(e)
```

Comment:
This should be `raise ServiceRequestFailed(e.message) from e` (and we have to hope that all of the exceptions have a message, but that's why we have tests...)

Reply:
`AttributeError: 'JSONDecodeError' object has no attribute 'value'` nor
AttributeError: 'JSONDecodeError' object has no attribute 'message'

Reply:
Then maybe `raise ServiceRequestFailed(str(e)) from e`

Reply:
ok done